### DISCUSSION TITLE
Use iframe in menu items?

### DISCUSSION THREAD

**Discussion by davidleitch1** [quarto-dev/quarto-cli#8941] [Q&A] [answered]
### How to use iframe as a menu item?

I run two websites and cohost a specialist podcast. One, the main website, is now built with Quarto and what a great technology,  and the other using Holoviz Panel.  The podcast is hosted by my cohost on his website. So the Quarto site has _quarto.yml:

    left: 
      - text: "NEM-dash"
        href: http://www.zzz.com
      - text: "Articles"
        file: posts.qmd
      - text: "Daily-jot"
        file: jotter.qmd
      - text: "Energy Insiders Podcast"

I would like to replace the href links with iframes but it doesn't work. On render it hangs. Is it possible to have an iframe as a menu link? Apologise for space cadet question, html and YAML are not my strong suits.

**Comment by mcanouil** [comment, marked as answer]
An iframe is a content.
A menu is a set of actual links.

It will never "work" in Quarto or anywhere.

If you want to embed a content from another website:
1. Simply add a link to the website.
2. Make a link to a page in which you embedded the website using iframe.

**Comment by davidleitch1** [comment]
Ok thanks.
